modules/mono/glue/GodotSharp/Godot.SourceGenerators.Internal/UnmanagedCallbacksGenerator.cs

```diff
@@ -73,8 +73,8 @@ private void GenerateAttribute(GeneratorPostInitializationContext context)
     {
         string source = @"using System;
 
-namespace Godot.SourceGenerators.Internal
-{
+namespace Godot.SourceGenerators.Internal;
```

Comment:
I wouldn't do these as the source generator output has to be compiled in the context of a user project, which might target an older C# version (tho this might already not be possible anyways)

Reply:
Oh! I thought it was in the same vein as the source generation for GodotSharp and GodotSharpEditor, my mistake